ProcessMaker/processmaker
# FOUR-18360 | Write Test Cases to Cover Scenarios for Applying CSS, Layout, Fields, and the Complete Template

# Feature
Ticket: [FOUR-18360](https://processmaker.atlassian.net/browse/FOUR-18360)
This PR adds unit tests for new Screen Templates functionality. The tests ensure that users can:
	- Apply CSS to an existing screen from a Screen Template
	- Apply Layout to an existing screen from a Screen Template
	- Apply Fields to an existing screen from a Screen Template
	- Apply a full Screen Template to an existing screen

# How to Test
1. Go to branch `subtask/FOUR-18360` in `processmaker`.
2. Run unit tests for the following tests in ScreenTemplateTest.php
	- `testApplyCssToExistingScreen()`
	- `testApplyFieldsToExistingScreen()`
	- `testApplyLayoutToExistingScreen()`
	- `testApplyFullTemplateToExistingScreen()`
3. All tests should pass

ci:processmaker:observation/FOUR-19607

# Code Review Checklist


[FOUR-18360]: https://processmaker.atlassian.net/browse/FOUR-18360?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
